packages/cisco_secure_endpoint/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.1.1"
```

To follow semantic versioning, a `type: enhancement` would usually mean version 2.2.0.

```suggestion
- version: "2.2.0"
```